cwms-data-api/src/main/java/cwms/cda/formatters/json/JsonV1.java

```diff
@@ -155,7 +154,9 @@ private Object buildFormatting(List<? extends CwmsDTOBase> daoList) {
                 // If dataType annotated with the class we can return an array of them.
                 // If a class needs to be handled differently an else_if branch can be added above
                 // here and a wrapper obj used to format the return value however is desired.
-                retVal = daoList;
+                // Added toArray due to serialization issues with List for TimeSeriesProfile.
```

Comment:
comment should specifically call out Jackson subclassing annotations, not `TimeSeriesProfile` (also it was the profile parser if we're being super accurate)